src/langserver.ts

```diff
@@ -40,8 +40,9 @@ export async function checkVlsInstallation(): Promise<boolean> {
 function receiveLauncherJsonData(cb: (d: { error?: { code: number, message: string }, message: string }) => void) {
 	return (rawData: string | Buffer) => {
 		const data = typeof rawData == 'string' ? rawData : rawData.toString('utf8');
-		log(`[v ls] new data: ${data}`);
-		cb(JSON.parse(data));
+		const escapedData = data.replaceAll('\\','/'); // replace backslashes found in Windows paths to prevent JSON parsing errors, TODO: proper JSON escaping solution needed
+		log(`[v ls] new data: ${data}\tescaped: ${escapedData}`);
+		cb(JSON.parse(escapedData));
```

Comment:
would be great to have this in a separate PR so we can easily rollback changes in case something goes wrong

Reply:
ok nevermind they are all connected lol